wiki/Difficulties/en.md

```diff
@@ -11,7 +11,22 @@ tags:
 
 *Not to be confused with [Beatmaps](/wiki/Beatmap).*
 
-A beatmap's **difficulty** describes the level of skill needed to complete it. The difficulty name and star rating of a beatmap usually give enough information to tell how difficult it is.
+A beatmap's **difficulty** describes the level of skill needed to complete it. The difficulty name and star rating usually give enough information to tell how challenging it is.
+
+## Overview
+
+A difficulty is stored as a text file with [the `.osu` extension](/wiki/osu!_File_Formats/Osu_(file_format)). It consists of [hit objects](/wiki/Hit_object) of a certain [game mode](/wiki/Game_mode), which are set up accordingly to the [timing](/wiki/Glossary#timing-section) tailored to a specific song. Difficulties have different aspects, which directly affect user experience.
```

Comment:
This description feels a bit odd. "Difficulties" should first be defined in a similar manner to how the beatmaps article described them. Plus "star rating" would prob fit this definition better, as they actually do tell you the level of skill needed to complete the difficulty.

Star rating should also be linked (https://osu.ppy.sh/wiki/en/Beatmapping/Star_rating)